src/main/java/org/folio/rest/impl/InvoiceStorageImpl.java

```diff
@@ -0,0 +1,90 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.Invoice;
+import org.folio.rest.jaxrs.model.InvoiceLine;
+import org.folio.rest.jaxrs.resource.InvoiceStorage;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static io.vertx.core.Future.succeededFuture;
+
+public class InvoiceStorageImpl implements InvoiceStorage {
+
+  public static final String NOT_SUPPORTED = "Not supported";
+
+  @Validate
+  @Override
+  public void getInvoiceStorageInvoices(int offset, int limit, String query, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    asyncResultHandler.handle(succeededFuture(GetInvoiceStorageInvoicesResponse.respond500WithTextPlain(NOT_SUPPORTED)));
```

Comment:
You can use `Response.status(501).build()` i.e. `501 (Not Implemented)` for now

Reply:
Done